website/versioned_docs/version-0.62/getting-started.md

```diff
@@ -0,0 +1,115 @@
+---
+id: version-0.62-getting-started
+title: Get Started with Windows
+original_id: getting-started
+---
+
+This guide will help you get started on setting up your very first React Native for Windows app.
+
+Make sure you have installed all of the [development dependencies](rnw-dependencies.md).
+
+If you're looking to build directly from the repo or you want to make contributions to the react-native-windows, check out the guide for [building the react-native-windows repo](building-rnw.md).
+
+For information around how to set up React Native, see the [React Native Getting Started Guide](https://reactnative.dev/docs/getting-started).
+
+## Install React Native for Windows
+
+Remember to call `react-native init` from the place you want your project directory to live.
+
+```
+npx react-native init <projectName> --version ^0.62.2
```

Comment:
`react-native-init` is to get the stock `react-native` instead of `react-native-windows`, so 0.62.2 is the latest stable built in RN. It's actually what we're testing against for `react-native-windows` 0.62 in repo.